setup.py

```diff
@@ -103,7 +103,12 @@ def read(filename):
         return f.read().decode('utf-8')
 
 
-dependencies = read('requirements.txt').split()
+dependencies = [
+    'click==6.7',
+    'psutil>=5.2.2,<6.0.0',
+    'requests>=2.5.0,<3.0.0',
```

Comment:
This is the requirement line as suggested in [this comment](https://github.com/pact-foundation/pact-python/pull/48#issuecomment-334966321).